src/metrics.jl

```diff
@@ -356,6 +357,27 @@ end
 end
 jaccard(a::AbstractArray, b::AbstractArray) = evaluate(Jaccard(), a, b)
 
+# BrayCurtis
+
+@inline eval_start(::BrayCurtis, a::AbstractArray{Bool}, b::AbstractArray{Bool}) = 0, 0
+@inline eval_start(::BrayCurtis, a::AbstractArray{T}, b::AbstractArray{T}) where {T} = zero(T), zero(T)
+@inline function eval_op(::BrayCurtis, s1, s2)
+    abs_m = abs(s1 - s2)
+    abs_p = abs(s1 + s2)
+    abs_m, abs_p
+end
+@inline function eval_reduce(::BrayCurtis, s1, s2)
+    @inbounds a = s1[1] + s2[1]
+    @inbounds b = s1[2] + s2[2]
+    a, b
+end
+@inline function eval_end(::BrayCurtis, a)
+    @inbounds v = a[1] / a[2]
```

Comment:
Should `a[1] == 0 && a[2] == 0` case give NaN or 0.0?

Reply:
It should give NaN (or throw an error) - rows where both counts are zero should not be included in the calculation.